Engines/Wine/Verbs/adobeair/script.js

```diff
@@ -1,4 +1,39 @@
+include(["engines", "wine", "engine", "object"]);
+include(["utils", "functions", "net", "resource"]);
+include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "verbs", "luna"]);
+
```

Comment:
Has it been tested?

Reply:
untested, forked from other verbs and my knowledge of winetricks and adobeair

Reply:
Please do not create untested pull requests. It doesn't help anyone because in the end it will not work.

Reply:
@plata Sorry, i was hoping to make it easier to finish this way..

Reply:
It's ok, if you need help or something like that. But in this case you should at least state that the script is untested and what you need to no. Otherwise we might merge it because we think it's finished and then we have broken scripts in the repository.

Reply:
At the end, it's not that hard to test it, and it would be very useful to have it. Can you do it?

Reply:
@plata Wanted to make that message, but hugged up the commit sorry about that again :X

Added info in commit message.

@qparis Have to go AFK now (i'm busy today) will test once i have the time.

Reply:
Testing

Reply:
More info how?

Reply:
Install it from the "Verb" pane on any container you have on your computer